lib/jobs/node-discovered-alert-job.js

```diff
@@ -43,16 +45,19 @@ function nodeAlertJobFactory(
      */
     NodeAlertJob.prototype._run = function run() {
         var self = this;
+        var nodeInfo = {};
         return waterline.nodes.needByIdentifier(self.nodeId)
         .then(function(node) {
             assert.string(node.type);
-
-            if(self.context.data){
-                return eventsProtocol.publishNodeEvent(node, 'discovered', self.context.data);
-            }
-            else{
-                return eventsProtocol.publishNodeEvent(node, 'discovered');
+            return lookup.findIpMacAddresses(self.nodeId)
+            .then(function(IpMacPairs) {
+               nodeInfo.IpMacAddress = IpMacPairs;
+               nodeInfo.nodeId = self.nodeId;
+               if(self.context.data){
+                  nodeInfo.contextData = self.context.data;
```

Comment:
The attribute name `contextData` is not suitable in Node Event, what kind of data is stored in the `context.data`? I think we could pick all nesting attribltes (in context.data) to the upper level (the same level with `nodeId` and `IpMacAddress`)